### DISCUSSION TITLE
Very high cpu usage on a virgin/iddle install

### DISCUSSION THREAD

**Discussion by fs30000** [crc-org/crc#3091] [Q&A] [answered]
Hello guys,

I'm installing crc for testing. On a virgin and idle install, the load averages inside crc (on the host machine too o course), are extremely high:
**load average: 129.51, 113.97, 107.26**

This is a laptop with a intel i5 1135G7 cpu and 16GB of RAM.

I have tried installing crc on these topologies:
-  Windows 10 > vmplayer > rhel8 > crc (12GB RAM to rhel8 and 11GB RAM to crc) *1
- rhel8 or fedora 35 > crc (14GB RAM to crc)

Details:
CodeReady Containers version: 2.0.1+bf3b1a6
OpenShift version: 4.10.3
Podman version: 3.4.4

Openshift is resource hungry, but is it supposed to be this hungry on a virgin and iddle install?

I want to thank in advance for any enlightenment about this. 

*1-  For the record, nested virtualization seems to be ok with vmware player, whereas with virtualbox  it does not.

**Comment by gbraad** [comment]
> Windows 10 > vmplayer > rhel8 > crc (12GB RAM to rhel8 and 11GB RAM to crc) *1
> *1- For the record, nested virtualization seems to be ok with vmware player, whereas with virtualbox it does not.

Nested virtualization can result in issues. We have seen similar reports around VirtualBox and therefore do not recommend running in a nested setup. Especially as CRC is resource heavy, it is advised not to introduce additional overhead. Any issue filed without a native virtualization stack is regarded as invalid. In case of Windows only Hyper-V is tested (the combination with nested KVM is known to work, but also not recommended)

**Comment by gbraad** [comment, marked as answer]
Duplicate of https://github.com/code-ready/crc/issues/707

**Reply by gbraad** [reply]
> rhel8 or fedora 35 > crc (14GB RAM to crc)

Can you detail more about the setup? Please continue the conversation on the noted dup issue?

**Reply by fs30000** [reply]
You can close this case.  Seems it's a generic kubernetes problem. Whatever instalation you do, cpu will skyrocket. Be it crc, openshift or k8s. I installed openshift in a powerful server. Load was stupidly high. Someone said this was by design. In a time everyone is talking about saving energy, this is lame and stupid.

**Reply by aaronsuns** [reply]
I have been using k3s for quite long time on my laptop, of course the key k8s component consume some CPU, but to use it is not a problem at all, could just let it run in background and do my other daily work at the same time.  
CRC is much much worse, after boot CRC, I'm not dare to open many windows, otherwise the whole computer could freeze any time.  with idle CRC, the CPU fan become very noisy and the CPU load is constantly around 400%.

Setup:
Fedora 39.
levono p14s, 8 core.
16GB RAM.
